<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoSysSettingsTreeBrowserForm</class>
 <widget class="QWidget" name="GeoSysSettingsTreeBrowserForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>189</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GeoModel volume hierachy:</string>
   </property>
  </widget>
  <widget class="VP1GeoTreeView" name="treeView_volumebrowser">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>32</y>
     <width>196</width>
     <height>91</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>214</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_close">
      <property name="text">
       <string>&amp;Close</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VP1GeoTreeView</class>
   <extends>QTreeView</extends>
   <header>VP1GeometrySystems/VP1GeoTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
